rosidl_typesupport_introspection_c/resource/msg__type_support.c.em

```diff
@@ -198,9 +199,15 @@ static rosidl_message_type_support_t @(function_prefix)__@(spec.base_type.type)_
   &@(function_prefix)__@(spec.base_type.type)_message_members
 };
 
-ROSIDL_TYPESUPPORT_INTROSPECTION_C_EXPORT_@(spec.base_type.pkg_name)
+ROSIDL_GENERATOR_C_EXPORT_@(spec.base_type.pkg_name)
```

Comment:
The c code generated in package `rosidl_typesupport_introspection_c` can not use this export macro. It must use its own.